grammar.js

```diff
@@ -232,12 +232,10 @@ module.exports = grammar({
 
     enum_body_elem: ($) =>
       choice(
-        seq($.id, "=", $.constant, optional($.deprecated)),
-        seq($.id, optional($.deprecated)),
```

Comment:
Nit: not sure we need multiple rules here when it all fits into one.

```javascript
    enum_body_elem: ($) =>
      seq($.id, optional(seq("=", $.constant)), optional($.attr)),
```

This is probably style, so feel free to drop if you disagree.